### ISSUE TITLE
Accessibility: Missing label for language selector

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#3358]
Branch: develop

Expected Behavior: Language selector combobox should have a descriptive label.
(WCAG 2.0 Level AA; Required)

Current Behavior: Form "select" control is without the appropriate label attribute.

![lang-select-aria-label](https://cloud.githubusercontent.com/assets/1457929/6744946/157a3828-ceb0-11e4-941e-5db85af2f681.png)

**Comment by radinamatic** [comment]
Solved in  #3466.
